schema/cadence/versioned/v0.2/add_signal_decision.cql

```diff
@@ -0,0 +1,9 @@
+CREATE TYPE signal_info (
```

Comment:
This should go into v0.3.  We have already released v0.2.

Reply:
Will move when rebase